flytekit/extras/tensorflow/records.py

```diff
@@ -0,0 +1,86 @@
+import pathlib
+from typing import Type, TypeVar
```

Comment:
Thank you @ryankarlos. Overall LGTM, could you put these files to `flytekit/extras/tensorflow/record/`